Sources/ConsulServiceDiscovery/ConsulServiceDiscovery.swift

```diff
@@ -1,7 +1,7 @@
 import Dispatch
-import ServiceDiscovery
+@preconcurrency import ServiceDiscovery
```

Comment:
I guess that it should be removed after migrating to ServiceDiscovery 2.0

Reply:
Yeah, maybe a #warning for it so we don't forget?

Reply:
Just to clarify: are you thinking of unconditional warning or about some check? [](edited-with-Axolo)

Reply:
I was just thinking of unconditional warning but open to better suggestions

Reply:
Hm, won't it be shown on every build even as a dependency? [](edited-with-Axolo)

Reply:
Done!